sopel/cli/plugins.py

```diff
@@ -0,0 +1,88 @@
+# coding=utf-8
+"""Sopel Plugins Command Line Interface (CLI): ``sopel-plugins``"""
+from __future__ import unicode_literals, absolute_import, print_function, division
+
+import argparse
+
+from sopel import plugins
+
+from . import utils
+
+
+def build_parser():
+    """Configure an argument parser for ``sopel-plugins``"""
+    parser = argparse.ArgumentParser(
+        description='Sopel plugins tool')
+
+    # Subparser: sopel-plugins <sub-parser> <sub-options>
+    subparsers = parser.add_subparsers(
+        help='Actions to perform',
```

Comment:
Can you actually specify more than one action at once? 😅

Because if not:

```suggestion
        help='Action to perform',
```